onnxruntime/python/tools/kernel_explorer/kernels/kernel_explorer.pyi

```diff
@@ -0,0 +1,7 @@
+class DeviceArray:
```

Comment:
Do we need this file?

Reply:
Yeah, otherwise pyright lint will dump garbarge in the Files changed tab